src/integTest/kotlin/io/snyk/plugin/snykcode/core/PduTest.kt

```diff
@@ -0,0 +1,32 @@
+package io.snyk.plugin.snykcode.core
+
+import com.intellij.openapi.util.io.FileUtil
+import com.intellij.openapi.vfs.LocalFileSystem
+import com.intellij.psi.PsiManager
+import com.intellij.testFramework.LightPlatformTestCase
+import com.intellij.testFramework.PlatformTestUtil
+import java.io.File
+
+class PduTest : LightPlatformTestCase() {
+
+    fun testOutOfProjectFile_shouldBeCorrectlyConvertingPath2PsiAndVs() {
+        val testFilePath = FileUtil.toSystemIndependentName(PlatformTestUtil.getCommunityPath()) + "/testFile"
+        File(testFilePath).createNewFile()
```

Comment:
For less manual cleanup, you could use File.createTempFile() and File.deleteOnExit() (see https://docs.oracle.com/javase/8/docs/api/java/nio/file/Files.html)

Reply:
It makes sense to use JUnit rule for TemporaryFile and Folder: https://junit.org/junit4/javadoc/4.12/org/junit/rules/TemporaryFolder.html

Reply:
Looks like deleteOnExit will not help us as it's javadocs says: "deleted when the virtual machine terminates" which has no sense in our case as we need it to be deleted after test method execution. So manual deletion needed here anyway. Thanks for pointing that out!

Reply:
I like Pavel's JUnit stuff. Dunno if it's available in JUnit 3 (`LightPlatformTestcase`) though.

Reply:
Just tried to do it through TemporaryFolder and for some reason neither `TemporaryFolder#before` (responsible for actual creation of temp folder) nor `TemporaryFolder#after` (responsible for deletion of all created temp files/dirs) are not executed. Any idea why that could be?...

Reply:
Seems like JUnit annotations (`@Rule` in our case) are ignored when run LightPlatformTestCase